src/DynamoCoreWpf/ViewModels/Preview/InfoBubbleViewModel.cs

```diff
@@ -465,6 +465,32 @@ public void OnRequestAction(InfoBubbleEventArgs e)
         /// </summary>
         /// <param name="dynamoViewModel"></param>
         public InfoBubbleViewModel(DynamoViewModel dynamoViewModel)
+        {
+            InitializeInfoBubble(dynamoViewModel);
+        }
+
+        /// <summary>
+        /// This constructor will update the TopBottom and BottomRight positions of the warnings since the beginning.
+        /// </summary>
+        /// <param name="dynamoViewModel"></param>
+        /// <param name="nodeViewModel">This parameter will be used to get the TopBottom and BottomRight positions of the NodeModel</param>
+        public InfoBubbleViewModel(NodeViewModel nodeViewModel)
+        {
+
+            InitializeInfoBubble(nodeViewModel.DynamoViewModel);
+
+            if (nodeViewModel != null)
```

Comment:
Should we move this check above the line above? I think param null check should be first of anything

Reply:
@QilongTang check moved one line above.
https://github.com/DynamoDS/Dynamo/pull/13015/commits/d18b0bc9afbe9f5abbdfefcbc8198dc32622b2ca